examples/demo.ipynb

```diff
@@ -30,7 +30,7 @@
    "cell_type": "markdown",
```

Comment:
Let's hold off updating this notebook until the corresponding version in `cleanvision-examples/` is merged. Then you can just clear the outputs of that one and merge it here (no review needed)